cardinal_toe_app/src/app/interfaces/Location.ts

```diff
@@ -0,0 +1,20 @@
+export interface Location {
+
+    //Required for UI
+    name: string,
+    description: string,
+
+    //Required for MapGeneratorService
+    previousLocation: Location | null,
+    isRootLocation: boolean,
+    isFinalLocation: boolean,
+    routes: Location[],
+    deviationValue: number
```

Comment:
Not a strict requirement here, but interfaces can extend other interfaces. Perhaps this could be described by a `GeneratedMapInterface` (or a better name)

Reply:
I understand how this could be useful but I think for the sake of potentially using these data for future features it might be best to leave as is for now?